src/FerriteGmsh.jl

```diff
@@ -161,16 +164,27 @@ function saved_file_to_grid(filename::String; domain="")
         # error code 2 is "no such file or directory".
         throw(SystemError("opening file $(repr(filename))", 2))
     end
-    gmsh.initialize()
+    should_finalize = Gmsh.initialize()
     gmsh.open(filename)
     fileextension = filename[findlast(isequal('.'), filename):end]
     dim = Int64(gmsh.model.getDimension()) # dont ask..
-    facedim = dim - 1
 
     if fileextension != ".msh"
         gmsh.model.mesh.generate(dim)
     end
- 
+    grid = to_grid(; domain=domain)
+    should_finalize && Gmsh.finalize()
+    return grid
+end
+
+"""
+    to_grid(; domain="")
+
+Generate a `Ferrite.Grid` from the current active/open model in the Gmsh library.
+"""
+function to_grid(; domain="")
```

Comment:
Thoughts on this name? Perhaps `model_to_grid` to follow the `saved_file_to_grid`? Although, to be honest, `saved_file_to_grid` isn't so good either.... Perhaps both can be called `to_grid`, and then have two methods, one for `::String` and one without.

Reply:
what do you think about `to_grid`? I'm asking because of the other functions in the src which do not have the underscore. But maybe you intended it, since they are more internal functions? Otherwise I'd vote for one consistent way for all functions

Reply:
I'm also in favor for having just two dispatches instead of two separate names